utils/fstack.c

```diff
@@ -327,6 +338,17 @@ static int setup_fstack_filters(struct uftrace_data *handle, char *filter_str,
 		pr_dbg("setup caller filters for %d function(s)\n", count);
 	}
 
+	if (hide_str) {
+		setting->info_str = hide_str;
+		walk_sessions(sessions, setup_hide, setting);
+		walk_sessions(sessions, count_filters, &count_hide);
```

Comment:
I don't think we can use count_filters here.  It just counts number of any entries, we need to count number of entries having the hide trigger.

Reply:
Please let me know if the updated patch is not what you wanted.